cli/template/extras/src/env/with-auth-db-planetscale.js

```diff
@@ -0,0 +1,60 @@
+import { createEnv } from "@t3-oss/env-nextjs";
+import { z } from "zod";
+
+export const env = createEnv({
+  /**
+   * Specify your server-side environment variables schema here. This way you can ensure the app
+   * isn't built with invalid env vars.
+   */
+  server: {
+    DATABASE_URL: z.string().url(),
```

Comment:
what's your thought on swapping out the url for host, username and password? I tend to prefer that as it's easier to switch between different users but idk if it's just me

Reply:
just for planetscale or also for other DBs where this can be possible (i think everything except sqlite?)

Reply:
personally if im swapping locally i usually just keep two connection strings in my env file and comment one out. i don't think i would find it more convenient, but not less convenient either. maybe worse for DB providers that only give connection strings so users would have to extract host/user/pw?

dont feel strongly either way, feel free to make the change if you think it's better :)

Reply:
Mainly planetscale maybe? Their docs suggests it: ![image](https://github.com/t3-oss/create-t3-app/assets/51714798/4c440030-9c63-4709-8a51-86c12f8c7524) ![image](https://github.com/t3-oss/create-t3-app/assets/51714798/dd507b2b-78f7-4bea-a9f3-121e02f5c248)

No need for the ssl query param note then too